sparkbox/designsystemssurvey
# Plural redirect

This redirects people who type in the singular version of our web address to the plural version.

To test:
This one takes a bit more to test. 
1. add `127.0.0.1    designsystemsurvey.seesparkbox.com` to your `etc/hosts` file
2. In the `server.js` file, change line 26 to read `if (host === 'designsystemsurvey.seesparkbox.com:3000') {`
3. Run the site
4. in the browser, go to `designsystemsurvey.seesparkbox.com:3000`

This should redirect you to the plural site address.
If you have any problems where `designsystemsurvey.seesparkbox.com:3000` doesn't load, you may have to comment out lines 9-12 (the SSL bit) and restart the app.
